modules/social_features/social_event/modules/social_event_an_enroll/src/EventAnEnrollOverride.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\social_event_an_enroll;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+
+/**
+ * Class EventAnEnrollOverride.
+ *
+ * Override event form.
+ *
+ * @package Drupal\social_event_an_enroll
+ */
+class EventAnEnrollOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * Returns config overrides.
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+    $config_factory = \Drupal::service('config.factory');
+
+    // Add field_event_an_enroll to event form.
+    $config_name = 'core.entity_form_display.node.event.default';
+    if (in_array($config_name, $names)) {
+      $config = $config_factory->getEditable($config_name);
+
+      $third_party = $config->get('third_party_settings');
+      $third_party['field_group']['group_event_visibility']['children'][] = 'field_event_an_enroll';
+
+      $content = $config->get('content');
```

Comment:
I think it's not possible to avoid overriding of all `content` property since we need to add new element right to under it. See also `SocialEventTypeConfigOverride`  and `SocialEventManagersConfigOverride`